R/derive_advs_params.R

```diff
@@ -0,0 +1,633 @@
+#' Adds a Parameter for Waist to Hip Ratio
+#'
+#' @description Adds a parameter for Waist to Hip Ratio using Waist Circumference and
+#' Hip Circumference for each by group (e.g., subject and visit) where the source parameters
+#' are available.
+#'
+#' **Note:** This is a wrapper function for the more generic [`admiral::derive_param_computed()`].
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` argument are expected to be in the dataset.
+#'   `PARAMCD`, and `AVAL` are expected as well.
+#'
+#'   The variable specified by `by_vars` and `PARAMCD` must be a unique key of the input dataset
+#'   after restricting it by the filter condition (`filter` argument) and to the parameters
+#'   specified by `wstcir_code` and `hipcir_code`.
+#'
+#' @param wstcir_code Waist Circumference parameter code
+#'
+#'   The observations where `PARAMCD` equals the specified value are considered
+#'   as the Waist Circumference.
+#'
+#'   *Permitted Values:* character value
```

Comment:
This is probably more applicable to admiral, so I would not change it here, but should this not be "character string" or just "character"?

Reply:
It's consistent with other admiral functions.